mirage/config.ts

```diff
@@ -420,4 +421,14 @@ export default function(this: Server) {
 
     // node analytics
     this.get('/metrics/query/node_analytics/:nodeID/:timespan', getNodeAnalytics);
+
+    // Addon service
+    this.urlPrefix = addonServiceUrl;
+    this.namespace = '/v1';
+    this.resource('external_storage_services', { only: ['index', 'show'] });
```

Comment:
Innnteresting. I didn't know `this.resource` was a thing.

Reply:
Yea, partway through writing some of these views, I realized it was a lot simpler than what I was doing 😅